src/strategies_and_params/poststepstrategy.jl

```diff
@@ -170,3 +170,13 @@ function loneliness(::Type{<:Complex}, vector, threshold)
     end
     return num_lonely / length(vector)
 end
+
+"""
+    Timer
```

Comment:
```suggestion
    Timer() <:  PostStepStrategy
```
Nice!

Should we mention this in the docstring of `PostStepStrategy`?

Reply:
Added to docstring.